docs/admin/users/oidc-auth/microsoft.md

```diff
@@ -0,0 +1,62 @@
+# Microsoft Entra ID authentication (OIDC)
+
+This guide shows how to configure Coder to authenticate users with Microsoft Entra ID using OpenID Connect (OIDC).
+
+## Prerequisites
+
+- A Microsoft Azure Entra ID Tenant
+- Permission to create Applications in your Azure environment
+
+## Step 1: Create an OAuth App Registration in Microsoft Azure
+
+1. Open Microsoft Azure Portal (https://portal.azure.com) → Microsoft Entra ID → App Registrations → New Registration.
+2. Name: Name your application appropriately
+3. Supported Account Types: Choose the appropriate radio button according to your needs. Most organizations will want to use the first one labeled "Accounts in this organizational directory only"
+4. Click on "Register"
+5. On the next screen, select: "Certificates and Secrets"
+6. Click on "New Client Secret" and under description, enter an appropriate description. Then set an expiry and hit "Add" once it's created, copy the value and save it somewhere secure for the next step.
+7. Next, click on the tab labeled "Token Configuration", then click "Add optional claim" and select the "ID" radio button, and finally check "upn" and hit "add" at the bottom.
+8. Then, click on the button labeled "Add groups claim" and check "Security groups" and click "Save" at the bottom.
```

Comment:
I think setting group claims is only required if you're using [IdP Sync, a premium feature](https://coder.com/docs/admin/users/idp-sync), but then that would also require setting `CODER_OIDC_GROUP_FIELD=groups` which isn't mentioned in Step 2's code block. 

I'd suggest either 
1. Add `CODER_OIDC_GROUP_FIELD=groups` to your Step 2 code block and a note explaining it's for Group Sync / IdP Sync OR
2. Make this an optional step and link back to the IdP Sync docs

For ref, the Google version of these docs doesn't make this callout